### ISSUE TITLE
Stacey absorbing boundaries

### ISSUE THREAD

**Issue by mpbl** [SPECFEM/specfem3d#50]
see with Zhinan if a spring should be added to Stacey in SPECFEM3D and SPECFEM3D_GLOBE; Stacey is a dashpot and thus Zhinan and his
advisors in China have shown that the condition can become inaccurate and have a static offset in some cases; they have shown how
to suppress this by adding a spring in parallel to the dashpot

already done in the 2D code

see with Zhinan if he thinks this should be done in SPECFEM3D and SPECFEM3D_GLOBE as well, and if it would be useful

should probably not be a high priority; PML and other things are more urgent

note DK DK Jan 2013: not needed any more because we now have C-PML instead; thus purposely not done.

(todo_list_please_dont_remove.txt: suggestion 20)
